diff --git a/dist/index.js b/dist/index.js
new file mode 100644
--- /dev/null	(date 1709108329289)
+++ b/dist/index.js	(date 1709108329289)
@@ -0,0 +1,1 @@
+export * from "./wa-sqlite-async.mjs";
diff --git a/dist/module.js b/dist/module.js
new file mode 100644
--- /dev/null	(date 1709108329291)
+++ b/dist/module.js	(date 1709108329291)
@@ -0,0 +1,1 @@
+export * from "./wa-sqlite-async.mjs";
diff --git a/package.json b/package.json
--- a/package.json	(revision dc9de5127b8db78f7c0d07875f8c3500b474d29a)
+++ b/package.json	(date 1709108329314)
@@ -1,7 +1,8 @@
 {
-  "name": "wa-sqlite",
+  "name": "@mendix/wa-sqlite",
   "version": "0.9.11",
   "type": "module",
+  "license": "MIT",
   "main": "src/sqlite-api.js",
   "types": "src/types/index.d.ts",
   "files": [
diff --git a/src/types/index.d.ts b/src/types/index.d.ts
--- a/src/types/index.d.ts	(revision dc9de5127b8db78f7c0d07875f8c3500b474d29a)
+++ b/src/types/index.d.ts	(date 1709108329303)
@@ -1053,7 +1053,7 @@
 }

 /** @ignore */
-declare module 'wa-sqlite/src/sqlite-constants.js' {
+declare module '@mendix/wa-sqlite/src/sqlite-constants.js' {
   export const SQLITE_OK: 0;
   export const SQLITE_ERROR: 1;
   export const SQLITE_INTERNAL: 2;
@@ -1286,8 +1286,8 @@
 }

 /** @ignore */
-declare module 'wa-sqlite' {
-  export * from 'wa-sqlite/src/sqlite-constants.js';
+declare module '@mendix/wa-sqlite' {
+  export * from '@mendix/wa-sqlite/src/sqlite-constants.js';

   /**
    * Builds a Javascript API from the Emscripten module. This API is still
@@ -1305,20 +1305,20 @@
 }

 /** @ignore */
-declare module 'wa-sqlite/dist/wa-sqlite.mjs' {
+declare module '@mendix/wa-sqlite/dist/wa-sqlite.mjs' {
   function ModuleFactory(config?: object): Promise<any>;
   export = ModuleFactory;
 }

 /** @ignore */
-declare module 'wa-sqlite/dist/wa-sqlite-async.mjs' {
+declare module '@mendix/wa-sqlite/dist/wa-sqlite-async.mjs' {
   function ModuleFactory(config?: object): Promise<any>;
   export = ModuleFactory;
 }

 /** @ignore */
-declare module 'wa-sqlite/src/VFS.js' {
-  export * from 'wa-sqlite/src/sqlite-constants.js';
+declare module '@mendix/wa-sqlite/src/VFS.js' {
+  export * from '@mendix/wa-sqlite/src/sqlite-constants.js';

   export class Base {
     mxPathName: number;
@@ -1433,7 +1433,7 @@
 }

 /** @ignore */
-declare module 'wa-sqlite/src/examples/ArrayModule.js' {
+declare module '@mendix/wa-sqlite/src/examples/ArrayModule.js' {
   export class ArrayModule {
     /**
      * @param {SQLiteAPI} sqlite3
@@ -1542,8 +1542,8 @@
 }

 /** @ignore */
-declare module 'wa-sqlite/src/examples/ArrayAsyncModule.js' {
-  import { ArrayModule } from "wa-sqlite/src/examples/ArrayModule.js";
+declare module '@mendix/wa-sqlite/src/examples/ArrayAsyncModule.js' {
+  import { ArrayModule } from "@mendix/wa-sqlite/src/examples/ArrayModule.js";
   export class ArrayAsyncModule extends ArrayModule {
     /**
      * @param {function} f
@@ -1554,8 +1554,8 @@
 }

 /** @ignore */
-declare module 'wa-sqlite/src/examples/IndexedDbVFS.js' {
-  import * as VFS from "wa-sqlite/src/VFS.js";
+declare module '@mendix/wa-sqlite/src/examples/IndexedDbVFS.js' {
+  import * as VFS from "@mendix/wa-sqlite/src/VFS.js";
   export class IndexedDbVFS extends VFS.Base {
     /**
      * @param {string} idbName Name of IndexedDB database.
@@ -1607,8 +1607,8 @@
 }

 /** @ignore */
-declare module 'wa-sqlite/src/examples/MemoryVFS.js' {
-  import * as VFS from "wa-sqlite/src/VFS.js";
+declare module '@mendix/wa-sqlite/src/examples/MemoryVFS.js' {
+  import * as VFS from "@mendix/wa-sqlite/src/VFS.js";
   export class MemoryVFS extends VFS.Base {
     name: string;
     mapNameToFile: Map<any, any>;
@@ -1617,14 +1617,14 @@
 }

 /** @ignore */
-declare module 'wa-sqlite/src/examples/MemoryAsyncVFS.js' {
-  import { MemoryVFS } from "wa-sqlite/src/examples/MemoryVFS.js";
+declare module '@mendix/wa-sqlite/src/examples/MemoryAsyncVFS.js' {
+  import { MemoryVFS } from "@mendix/wa-sqlite/src/examples/MemoryVFS.js";
   export class MemoryAsyncVFS extends MemoryVFS {
   }
 }

 /** @ignore */
-declare module 'wa-sqlite/src/examples/tag.js' {
+declare module '@mendix/wa-sqlite/src/examples/tag.js' {
   /**
    * Template tag builder. This function creates a tag with an API and
    * database from the same module, then the tag can be used like this:
